common/modules/gcp-project/main.tf

```diff
@@ -231,6 +263,15 @@ locals {
               "/^\\./",
               "")
             }"
+
+  service_account = "${element(concat(google_service_account.project.*.email, list("")),0)}"
+
+  # stg, prd, dev, and dev-doe are managed by the CI so they should have the service account
+  # and the permissions attached to it
+  #
+  # I wish Go supports this expression instead (?!^dev-doe$)(^dev-.*), but "?!" is not valid.
+  #
+  root_project_iam = "${replace(var.project_name, "/^dev-doe$/", "") != "" && replace(var.project_name, "/^dev-.*/", "") == ""}"
```

Comment:
hardcoded `dev-doe` here is quite ugly... can't we just use service account or smth for that project? what about the `serpan@` account? :)